test/tap/version-from-git.js

```diff
@@ -142,7 +142,7 @@ test('npm version from-git with an existing version', function (t) {
   }
 
   function checkVersion (er) {
-    t.equal(er.message, 'Version not changed')
+    t.ok(er.message.match(/Version not changed.*/))
```

Comment:
tap has a verb for doing regexp compares, using that this would look like:
```
t.like(er.message, /Version not changed/)
```
(And the trailing `.*` is uneccessary)